exercises/regularization/ex_rnw/ex_lasso_subdifferential.Rnw

```diff
@@ -17,16 +17,16 @@ Optimization routines for the Lasso use coordinate gradient descent, but instead
           c(expression(x^2), expression(2*x^2), expression(paste("max(0,", 0.8*x, ")")), "|x|"))
 @
 
-  \item A subdifferential of $f$ is a set of values $\breve{\nabla}_x f$ defined as $$\breve{\nabla}_x f = \{ g: f(x) \geq f(x_0) + g \cdot (x-x_0) \, \forall x \in \mathcal{I} \}.$$ Every scalar value $g \in \breve{\nabla}_x$ is said to be a subgradient of $f$.  Does a subdifferential have any parallels to the previous question? How can we interpret $g$?
+  \item A subdifferential of $f$ is a set of values $\breve{\nabla}_{x_0} f$ defined as $$\breve{\nabla}_{x_0} f = \{ g: f(x) \geq f(x_0) + g \cdot (x-x_0) \, \forall x \in \mathcal{I} \}.$$ Every scalar value $g \in \breve{\nabla}_{x_0}$ is said to be a subgradient of $f$.  Does a subdifferential have any parallels to the previous question? How can we interpret $g$?
```

Comment:
"is said to be a subgradient of f *at x_0*"

Reply:
updated with last commit fe8dc3a11d71afcc3d170f2e580b71ffba1c691b